scss/index.scss

```diff
@@ -2,16 +2,17 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-  width: 1000px;
+  padding: 75px 200px;
 }
 
 .list-item {
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
+  flex: 1 0 20%;
```

Comment:
why flex-basics 20%?

Reply:
Because I thought that if I used this percentage, the spaces would be distributed